static/src/stylesheets/module/email/_form.scss

```diff
@@ -0,0 +1,207 @@
+// TODO These mixins are here while we run the A/B test as the styles are custom to the email form but if they become perm then
+// they should be made reusable
+
+@mixin rounded-input--inline-button() {
+    &text-input {
+        border-right: 0;
+        @include border-radius(1000px 0 0 1000px);
+        float: left;
+        width: 65%;
+    }
+
+    &submit-input {
+        font-size: 12px;
+        @include border-radius(0 1000px 1000px 0);
+        border-left: 0;
+        float: left;
+        margin-right: 0;
+        padding-left: 10px;
+        width: 35%;
+    }
+}
+
+.email-sub {
+    @include clearfix();
+}
+
+.email-sub__iframe {
+    width: 100%;
+}
+
+.email-sub__form {
+    font-family: $f-sans-serif-text;
+    font-size: 14px;
+    padding: 20px;
+
+    * {
+        box-sizing: border-box;
+    }
+
+    &--is-hidden {
+        display: none;
+    }
+}
+
+.email-sub__label {
+    color: $form-primary-colour;
+
+    &--is-hidden {
+        visibility: hidden;
+    }
+}
+
+// Set with JS so we can use a real label
+// instead of abusing the placeholder attr
+// but displays correctly if JS doesn't load
+.email-sub__inline-label--enabled {
+    position: relative;
+    width: 100%;
+
+    .email-sub__label {
+        color: guss-colour(neutral-2, $pasteup-palette);
+        line-height: 3;
+        left: 15px;
+        position: absolute;
+        pointer-events: none; // Ensure the text input cursor is shown
+    }
+}
+
+.email-sub__text-input {
+    border: 1px solid $form-primary-colour;
+    color: guss-colour(neutral-2, $pasteup-palette);
+    height: 3em;
+    outline: none;
+    padding: 8px 15px;
+}
+
+.email-sub__submit-input {
+    border-color: $form-secondary-colour;
+    color: $form-primary-colour;
+
+    &:focus,
+    &:hover {
+        border-color: $form-primary-colour;
+    }
+}
+
+.email-sub__text-input,
+.email-sub__submit-input {
+    margin-bottom: 15px;
+    width: 100%;
+}
+
+.email-sub__small {
+    color: $form-secondary-colour;
+    display: block;
+}
+
+.email-sub__message {
+    color: #ffffff;
+    padding-left: 35px;
+    position: relative;
+
+    &:before {
+        @include icon(close-icon-white-small);
+        content: '';
+        position: absolute;
+        height: 30px;
+        width: 30px;
+        bottom: 0;
+        left: 0;
+        top: 0;
+        margin: auto;
+    }
+}
+
+.email-sub__message__headline,
+.email-sub__message__description {
+    font-family: $f-sans-serif-text;
+}
+
+.email-sub__message__headline {
+    font-size: 14px;
+    font-weight: bold;
+    margin-bottom: 0;
+}
+
+.email-sub__message__description {
+    font-size: 12px;
+}
+
+
+
+/**
+* Modifications
+*/
+
+    // FOOTER MODS
+.email-sub__form--footer {
+    margin-bottom: $gs-baseline;
+    padding: 0 20px;
```

Comment:
`$gs-baseline` again (will stop marking them all - give us a shout if it doesn't make sense)